### ISSUE TITLE
Include a Identity Standard

### ISSUE THREAD

**Issue by thepiwo** [aeternity/aepp-base#442]
- using naming system
- e.g. include profile picture or other sharable data
- should be optional
- maybe public at beginning for obvious public data

**Comment by j28** [comment]
@thepiwo is this potentially a mix of 2 different stories? The first one being æternity defining an identity standard on the protocol level. The second one being the Base aepp implementing support for this identity standard.

If the above is the case, the 2nd story is highly dependent on the first. If no (there is no such dependency), the Base aepp team has much more freedom on what to implement for handling identities.

**Comment by thepiwo** [comment]
Yes it is, sure they heavily depend on each other. Specifying is probably not a task for the base-aepp.

**Comment by j28** [comment]
Got it. I updated your comment which in this case acts as the description of the issue. We can also move it to be the description, if you prefer this. Thanks!
